docs/docs-content/enterprise-version/upgrade/upgrade-vmware/non-airgap.md

```diff
@@ -8,13 +8,14 @@ tags: ["palette", "self-hosted", "vmware", "non-airgap", "upgrade"]
 keywords: ["self-hosted", "enterprise"]
 ---
 
-This guide takes you through the process of upgrading a self-hosted Palette instance installed on VMware vSphere.
+This guide takes you through the process of upgrading a self-hosted Palette instance installed on VMware vSphere. Before
+upgrading Palette to a new major version, you must first update it to the latest patch version of the latest minor
+version available. Refer to the [Supported Upgrade Paths](../upgrade.md#supported-upgrade-paths) section for details.
 
 :::warning
 
-Before upgrading Palette to a new major version, you must first update it to the latest patch version of the latest
-minor version available. Refer to the [Supported Upgrade Paths](../upgrade.md#supported-upgrade-paths) section for
-details.
+If you are upgrading from a Palette version that is older than 4.4.14, ensure that you have executed the utility script
```

Comment:
Let's link to your awesome troubleshooting guide, just to help readers land on this page.